tools/topology/platform/common/ssp.m4

```diff
@@ -38,6 +38,9 @@ define(`SSP_CONFIG_DATA',
 `	tuples."short" {'
 `		SOF_TKN_INTEL_SSP_MCLK_ID'	ifelse($4, `', "0", STR($4))
 `	}'
+`	tuples."word" {'
+`		SOF_TKN_INTEL_SSP_QUIRKS'	ifelse($5, `', "0", STR($5))
```

Comment:
@fredoh9 still you can merge this with the previous word filed. The id is not matter.

Reply:
@xiulipan got it